Sources/hostmgr/helpers/S3.swift

```diff
@@ -89,7 +89,8 @@ struct S3Manager {
             .value
         )
         let estimatedDownloadSpeedInMBPS: Int64 = 10
-        let timeout = totalMB / estimatedDownloadSpeedInMBPS
+        let minimalTimeout: Int64 = 60
+        let timeout = max(totalMB / estimatedDownloadSpeedInMBPS, minimalTimeout)
         logger.info("Download timeout: \(timeout / 60) minutes")
```

Comment:
If I read the code correctly, if `minimalTimeout` is the one picked at runtime, we might have a 1 minute timeout. 

1 minute, seems _very_ minimal, but maybe only to someone like me who doesn't have experience downloading large files from the S3 API.

Reply:
Ops, sorry I should've asked you to review the last commit before I merged this PR.

I thought about a case at last minute, where this function is used to download a relative small file. Adding a minimal timeout here to avoid the timeout gets too short. To give an extreme example, say we are about to download a 1 MB file, without this minimal timeout, the timeout value would be 0.1 second, which I believe is too short for establishing HTTP connection and transporting data back and forth. Having a minimal timeout value would make sure that there is enough time to download a relative small file, whose size is less than `minimalTimeout x estimatedDownloadSpeedInMBPS`(600 MB).

Reply:
> Ops, sorry I should've asked you to review the last commit before I merged this PR.

No worries! That was quite a small change.

> I thought about a case at last minute, where this function is used to download a relative small file.

That didn't occur to me, good point!